packages/plugin-ext/src/plugin/tabs.ts

```diff
@@ -325,10 +317,13 @@ export class TabsExtImpl implements TabsExt {
             return group;
         });
 
-        // Set the active tab group id
-        const activeTabGroupId = assertIsDefined(tabGroups.find(group => group.isActive === true)?.groupId);
-        if (activeTabGroupId !== undefined && this.activeGroupId !== activeTabGroupId) {
-            this.activeGroupId = activeTabGroupId;
+        const groupId = tabGroups.find(group => group.isActive === true)?.groupId;
+        // Set the active tab group id. skip if no tabgroups are open
+        if (tabGroups.length > 0) {
+            const activeTabGroupId = assertIsDefined(groupId);
+            if (activeTabGroupId !== undefined && this.activeGroupId !== activeTabGroupId) {
+                this.activeGroupId = activeTabGroupId;
+            }
         }
```

Comment:
this is seemingly done by VSCode to ensure that there allways is an active groupt, otherwise there is something broken. So so as far as i understood it this is more for the devoloper to show that there still is an issue

Reply:
I see, alright 👍